docs/core/get-started.md

```diff
@@ -1,51 +1,83 @@
 ---
 title: Get started with .NET Core
 description: Find resources to learn how to build .NET Core applications on Windows, Linux and macOS.
-author: johalex
-ms.author: johalex
-ms.date: 09/14/2017
+author: thraka
+ms.author: adegeo
+ms.date: 06/27/2018
 ---
+
 # Get started with .NET Core
 
-.NET Core runs on [Windows](#windows), [Linux](#linux), and [macOS / OS X](#os-x--macos).
+This article provides information on getting started with .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor while your code produces cross-platform libraries and applications. 
```

Comment:
"You can code in your favorite text editor while your code produces cross-platform libraries and applications. " I think using "code" in two different ways make this a bit awkward.  Maybe something like "You can write code in your favorite text editor that produces cross-platform libraries and applications."?